### DISCUSSION TITLE
Is there a way for MPV to receive commands even when it's not the focused window?

### DISCUSSION THREAD

**Discussion by 1000Fear** [mpv-player/mpv#15066] [Q&A] [answered]
Like pause, fast-forward, etc. Suppose I'm watching a tutorial on some software, I want to be able to operate the software simultaneously while watching. When I need to pause the tutorial video due to any issues, I would like to be able to do so simply by pressing a button, rather than having to switch to the MPV window to make adjustments. I have a Razer mouse with its driver, which can execute simple commands, and this might be helpful.

**Comment by guidocella** [comment, marked as answer]
https://mpv.io/manual/master/#json-ipc

**Reply by 1000Fear** [reply]
Thank you!
